### PR TITLE
Add additional instructions for repl Exercises

### PR DESCRIPTION
**Pull request by MicoPham**
The repl was read-only unless forked onto an existing account. 

Also added in bullet points under each exercise to give clarity on which file was to be worked on.

### CODE REVIEW

**Review by I3uckwheat — APPROVED** [review]

**Review by I3uckwheat — CHANGES_REQUESTED** [review]
Hey, we should be able to use standard markdown syntax here and avoid HTML. Thank you for this change, I was getting 20 emails a day on this and hadn't had time to investigate.

**Comment by MicoPham** [comment]
Hopefully I did it right this time.

File: foundations/javascript_basics/fundamentals-2.md [hunk 1]
```diff
@@ -48,11 +48,17 @@ To give you a good bit of practice, we have created some repl.it exercises for y
...
+
 - [Exercise 1](https://repl.it/@I3uckwheat/troubleshooting#troubleshooting.js)
+  *In this excercise, you will be working out of the file called trobuleshooting.js
 - [Exercise 2](https://repl.it/@I3uckwheat/enter-a-number#script.js)
+  *You will be working out of script.js 
 - [Exercise 3](https://repl.it/@I3uckwheat/lets-do-some-math#math.js)
+  *You will be working out of math.js  
 - [Exercise 4](https://repl.it/@I3uckwheat/direction-follow#follow.js)
-
+  *You will be working out of follow.js
```
Comment (I3uckwheat) [inline foundations/javascript_basics/fundamentals-2.md:61]: Not quite, you'll need a space after the `*`.

Feel free to play around with our new preview tool as well: https://www.theodinproject.com/lessons/preview

**Review by I3uckwheat — CHANGES_REQUESTED** [review]

File: foundations/javascript_basics/fundamentals-2.md (hunk 1 above)
Comment (MicoPham) [reply foundations/javascript_basics/fundamentals-2.md]: I was blindly guessing at what I was doing, but that preview tool is VERY helpful. Thanks alot Briggs !

Comment (I3uckwheat) [reply foundations/javascript_basics/fundamentals-2.md]: Thank YOU for doing this :)

**Pull request merged**